draft-kazuho-httpbis-priority.md

```diff
@@ -549,6 +555,17 @@ reflecting the urgencies expressed in the requests, so that less urgent
 responses would receive less bandwidth in case the bottleneck exists between the
 server and the intermediary.
 
+### HTTP/1 origins {#http11}
+
+It is common in CDN infrastructure to support HTTP/2 or HTTP/3 at the client
```

Comment:
```suggestion
It is common in CDN infrastructure to support HTTP/2 or HTTP/3 at the client-
```

Reply:
but also I might say something more generic like 

it is common for CDN infrastracture to support different HTTP versions on the front end and back end. For instance, the client-facing edge might support HTTP/2 and HTTP/3 while communication to origin servers is done using HTTP/1.1.

Reply:
Agreed, will resolve.